tests/unit/lib/utils/test_tar.py

```diff
@@ -19,7 +19,10 @@ def test_generating_tarball(self, temporary_file_patch, tarfile_open_patch):
 
         tarfile_file_mock.add.assert_called()
         tarfile_file_mock.add.assert_has_calls(
-            [call("/some/path", arcname="/layer1"), call("/some/dockerfile/path", arcname="/Dockerfile")],
+            [
+                call("/some/path", arcname="/layer1", filter=None),
+                call("/some/dockerfile/path", arcname="/Dockerfile", filter=None),
+            ],
```

Comment:
It looks like these tests only cover the case with `filter=None`, can we add test case for the new use case?

Reply:
Added test case with filter.